components/Tabs/src/MDCTabBar.m

```diff
@@ -268,8 +275,15 @@ - (CGSize)intrinsicContentSize {
   return _itemBar.intrinsicContentSize;
 }
 
+
 - (CGSize)sizeThatFits:(CGSize)size {
-  return [_itemBar sizeThatFits:size];
+  size = [_itemBar sizeThatFits:size];
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    size.height += self.safeAreaInsets.bottom;
```

Comment:
This should probably be conditional on the bar position? I think it wouldn't make sense to add the bottom insets for top tabs (and vice-versa).

Reply:
self.safeAreaInsets is conditional on the bar position

Reply:
It's possible for a parent VC to set safeAreaInsets that overlap the tabs even if they aren't in the bottom position. In that case, does it still make sense to increase the height?

Reply:
I don't know, you mean by setting additionalSafeAreaInsets? What would be the use case? Since this is a tab bar, and it has to account for the "normal" safe area inset at the bottom for the home indicator (if it's in that position), I'd say yes, it has to expand to account for any safe area inset. If the parent VC wants to add even more, I don't see how that'd change this behavior.

Reply:
If the tabbar were scrolling up along with some content, wouldn't the safeAreaInsets be set?  Sorry for any confusion, I was trying to back-up @brianjmoore and his suggestion to check the bar position.

Reply:
I'm not following the use case you're describing. So to answer the original suggestion again: "This should probably be conditional on the bar position?" -> yes, it should and it is, as safeAreaInsets is already conditional on the position.

Reply:
Discussed offline, I'll add a check using isTopTabsForPosition just as extra safety measure I guess